cfme/tests/webui/test_general_ui.py

```diff
@@ -226,6 +227,28 @@ def test_infrastructure_filter_20k_vms(appliance, create_20k_vms):
     assert items_amount >= 20000, 'Vms count is less than should be filtered'
 
 
+@pytest.mark.ignore_stream("5.10")
+@pytest.mark.tier(2)
+def test_welcoming_page(appliance, has_no_providers):
```

Comment:
I think this should be taken a step further and parameterized for all the compute types.

Reply:
@jawatts The idea is good, but I think it would be better to write a separate test for it. This test is about checking the welcome page that the user first sees on login when there are no providers.

Reply:
@jawatts That is a good idea, but this test checks the view user first sees when they login into an appliance that has no providers. I think parametrizing the test would slightly deviate from the case that we're trying to test. But I can add a separate test for this. Let me know what you think.

Reply:
@valaparthvi that sounds good to me. If you could add a `TODO` somewhere, that be great.

Reply:
Cool! I'll add a new test case along with this.

Reply:
@jawatts I have added the test, you can review it [here](https://github.com/ManageIQ/integration_tests/pull/8805/files#diff-b97f49d8a6659114ea4b051eb76698e1R269).